docker-compose.yml

```diff
@@ -77,12 +77,7 @@ services:
       - ATO_PUBLIC_KEY_FILE=../shared_files/ATO_public.pem
       - ATO_PRIVATE_KEY_FILE=../shared_files/ATO_private.pem
       - HTTP_ONLY=true
-      - OKTA_CLIENT_ORGURL=https://cms-sandbox.oktapreview.com
```

Comment:
These are for the SSAS container, don't we still need these values? It does look like some of the other ones (like BCDA_AUTH_PROVIDER) is not needed.